packages/tags/src/Tag.ts

```diff
@@ -77,9 +85,23 @@ export class Tag extends LitElement {
     }
 
     protected render(): TemplateResult {
+        const slots = [];
```

Comment:
I was having a bit a trouble getting the typechecking to pass for this line when running `yarn storybook`

Do you think we should set the type of `const slots` to `TemplateResult[]`?

Reply:
Not sure why I didn't run into this on the first pass, but I hit it in https://github.com/adobe/spectrum-web-components/pull/785/files#diff-f3e3a8af2e1fc7625253b6bae2053264R88 where the extended typing cleans it up. Could go either way on copying it over or prioritizing the Split Button PR to clear this up.

Reply:
I'm down for prioritizing the Split Button PR and resolving this during the rebase  👍